commit 4bdb45a8acfee805c715ca8ccf3cf74aac1a5223
Author: fryshorts <leonhard@in-verted.de>
Date:   Sun May 3 17:09:29 2015 +0200

    linux-v4l2: Add include directory to cmake
    
    Add the include directories found by cmake to the v4l2 plugin.
    This allows for the plugin to compile when the vl42 headers were
    found in a directory that is not normally in the search path of the
    compiler (e.g. /usr/local/include)

--- plugins/linux-v4l2/CMakeLists.txt
+++ plugins/linux-v4l2/CMakeLists.txt
@@ -24,6 +24,11 @@ else()
 	add_definitions(-DHAVE_UDEV)
 endif()
 
+include_directories(
+	SYSTEM "${CMAKE_SOURCE_DIR}/libobs"
+	${LIBV4L2_INCLUDE_DIRS}
+)
+
 set(linux-v4l2_SOURCES
 	linux-v4l2.c
 	v4l2-input.c
